dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/liberty/CDIRuntimeImpl.java

```diff
@@ -92,6 +93,7 @@
     private final AtomicServiceReference<InjectionEngine> injectionEngineServiceRef = new AtomicServiceReference<InjectionEngine>("injectionEngine");
     private final AtomicServiceReference<ScheduledExecutorService> scheduledExecutorServiceRef = new AtomicServiceReference<ScheduledExecutorService>("scheduledExecutorService");
     private final AtomicServiceReference<ExecutorService> executorServiceRef = new AtomicServiceReference<ExecutorService>("executorService");
+    private final AtomicServiceReference<ExecutorService> managedExecutorServiceRef = new AtomicServiceReference<ExecutorService>("managedExecutorService");
```

Comment:
I don't think so. It's working as written.

Reply:
Whether it works isn't the right way to decide.  Either interface will work.
ManagedExecutorService inherits from ExecutorService. It is made available in the service registry as both. Typically, components that want to avoid compile time dependencies on EE Concurrency and don't need any of the more advanced capability like ManagedTaskListener/execution properties will find it more convenient to  access it in terms of the Java SE ExecutorService interface.  For this reason, I would agree the code should remain as is here.